src/EnergyPlus/HVACManager.cc

```diff
@@ -970,127 +971,132 @@ void SimHVAC(EnergyPlusData &state)
 
                 for (AirSysNum = 1; AirSysNum <= NumPrimaryAirSys; ++AirSysNum) {
 
-                    if (any(state.dataConvergeParams->AirLoopConvergence(AirSysNum).HVACMassFlowNotConverged)) {
+                    auto & arrayRef = state.dataConvergeParams->AirLoopConvergence(AirSysNum).HVACMassFlowNotConverged;
+                    if (std::any_of(std::begin(arrayRef), std::end(arrayRef), [](bool i) {return i;})) {
```

Comment:
With arrays, we don't need to use the Array1D custom any function, we just need to use the std::algorithm any_of function.  For an array of bools, we just need a simple predicate that returns the value (I wish that was optional for bool types).

Reply:
lambda ftw.

Reply:
:+1: